02-toolbar/style.css

```diff
@@ -0,0 +1,232 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 10px;
+}
+body {
+    background-color: #223997;
+    height: 100vh;
+    position: relative;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    font-weight: 500;
+}
+
+.toolbar {
+    background-color: white;
+    width: 50rem;
+    height: 25rem;
+    position: relative;
+    top: 50vh;  
+    left: 50vw;
+    transform: translate(-50%, -50%);
+    border-radius: 0 0 10% 10%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+.toolbar__btns {
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+    height: 0vh;
+}
+
+.toolbar__btns .btn {
+    list-style: none;
+    position: relative;
+    width: 100px;
+    height: 50px;
+    /* clip-path: polygon(0 0, 100% 0, 100% 100%, 0 100%);
+    background: red; */   
+}
+
+.btn--label {
+    cursor: pointer;
+    position: relative;
+    top: 0; 
+    left: 0;
+    font-size: 1.9rem;
+    display: flex;
+}
+
+/* Icons positioning */
+
+.toolbar__btns .btn--one .icon::before, 
+.toolbar__btns .btn--two .icon::before {
+    left: 15px;
+}
+.toolbar__btns .btn--one .icon::after,
+.toolbar__btns .btn--two .icon::after {
+    left: 23px;
+}
+
+
+.toolbar__btns .btn--three .icon::before {
+    left: 7px;
+}
+.toolbar__btns .btn--three .icon::after {
+    left: 12px;
+}
+
+
+.toolbar__btns .btn--four .icon::before {
+    left: 22px;
+}
+.toolbar__btns .btn--four .icon::after {
+    left: 32px;
+}
```

Comment:
> 1. I centered each icon and ripple separately, there must be a better way? I tried flex and vertical-align

Since these elements are all position absolute, there isn't a way to center them all together. When centering stuff with position absolute you can use:

```
position: absolute;
top: 50%;
left: 50%;
transform: translate(-50%, -50%)
```

Before you do this make sure that:
- each item (li) is a perfect square with `position: relative`.
- each label has width and height 100% to occupy the whole item.

This will make it easier to center the elements. You can use the outline tip I explained to visually see stuff around.

Reply:
When applying this what I've noticed is the icons' left side is at the center of the label but the width still makes it off to the right. If that's what's supposed to happen (and not the result of something I've missed), I still need to tweak them individually?

Reply:
With just words is hard to understand what you mean. 🤔 A screenshot of what's wrong + code would help (if you don't want to commit/push it)

Reply:
Sorry I fixed it but forgot to update the comment :D

Reply:
I'd have to rewrite code to take a screenshot. Basically using 

top: 50%;
left: 50%;
transform: translate(-50%, -50%)

put the left side of each icon at the exact center of parent div, but what I want is the middle of each icon to be at the center. What seemed like the only way was to move each icon separately